sky/catalog/data_fetchers/fetch_runpod.py

```diff
@@ -0,0 +1,466 @@
+        'max_count': 8
+    },
+    'RTXPRO6000': {
+        'vcpus': 14.0,
+        'max_count': 8
+    },
+    'RTXPRO6000-MaxQ': {
+        'vcpus': 18.0,
+        'max_count': 8
+    },
+    'RTXPRO6000-WK': {
+        'vcpus': 32.0,
+        'max_count': 8
+    },
+    'V100-SXM2-32GB': {
+        'vcpus': 20.0,
+        'max_count': 8
+    }
+}
+
+# for backwards compatibility, force rename some gpus.
+# map the generated name to the original name
+GPU_NAME_OVERRIDES = {
+    'A100-PCIe': 'A100-80GB',
+    'A100-SXM': 'A100-80GB-SXM',
+    'H100-PCIe': 'H100',
+}
+
+# Constants
+USEFUL_COLUMNS = [
+    'InstanceType',
+    'AcceleratorName',
+    'AcceleratorCount',
+    'vCPUs',
+    'MemoryGiB',
+    'GpuInfo',
+    'Region',
+    'SpotPrice',
+    'Price',
+    'AvailabilityZone',
+]
+
+# Mapping of regions to their availability zones
+REGION_ZONES = {
+    'CA': ['CA-MTL-1', 'CA-MTL-2', 'CA-MTL-3'],
+    'CZ': ['EU-CZ-1'],
+    'IS': ['EUR-IS-1', 'EUR-IS-2', 'EUR-IS-3'],
+    'NL': ['EU-NL-1'],
+    'NO': ['EU-SE-1'],
+    'RO': ['EU-RO-1'],
+    'SE': ['EU-SE-1'],
+    'US': [
+        'US-CA-1',
+        'US-CA-2',
+        'US-DE-1',
+        'US-GA-1',
+        'US-GA-2',
+        'US-IL-1',
+        'US-KS-1',
+        'US-KS-2',
+        'US-NC-1',
+        'US-TX-1',
+        'US-TX-2',
+        'US-TX-3',
+        'US-TX-4',
+        'US-WA-1',
+    ],
+}
+
+
+def get_gpu_details(gpu_id: str, gpu_count: int = 1) -> Dict:
+    """Get detailed GPU information using GraphQL query.
+
+    This uses a custom graphql query because runpod.get_gpu(id) does not include
+    full lowestPrice information.
+    """
+    query = f"""
+    query GpuTypes {{
+      gpuTypes(input: {{id: "{gpu_id}"}}) {{
+        maxGpuCount
+        id
+        displayName
+        manufacturer
+        memoryInGb
+        cudaCores
+        secureCloud
+        communityCloud
+        securePrice
+        communityPrice
+        oneMonthPrice
+        threeMonthPrice
+        oneWeekPrice
+        communitySpotPrice
+        secureSpotPrice
+        lowestPrice(input: {{gpuCount: {gpu_count}}}) {{
+          minimumBidPrice
+          uninterruptablePrice
+          minVcpu
+          minMemory
+          stockStatus
+          compliance
+          maxUnreservedGpuCount
+          availableGpuCounts
+        }}
+      }}
+    }}
+    """
+
+    result = graphql.run_graphql_query(query)
+
+    if 'errors' in result:
+        raise RuntimeError(f'GraphQL errors: {result["errors"]}')
+
+    try:
+        gpu_type = result['data']['gpuTypes'][0]
+    except Exception as e:
+        error_msg = ('No GPU Types found in RunPod query with'
+                     f'gpu_id={gpu_id}, gpu_count={gpu_count}')
+        raise ValueError(error_msg) from e
+
+    return gpu_type
+
+
+def format_price(price: float) -> float:
+    """Format price to two decimal places."""
+    return round(price, 2)
+
+
+def format_gpu_name(gpu_type: Dict[str, Any]) -> str:
```

Comment:
It may be helpful to centralize this helper somewhere. I don't know how the rest of the repo works but this feels like something you may want to centralize